pro/src/design-system/TextInput/TextInput.spec.tsx

```diff
@@ -0,0 +1,94 @@
+import { render, screen } from '@testing-library/react'
+import { axe } from 'vitest-axe'
+
+import { TextInput, type TextInputProps } from './TextInput'
+
+const defaultProps: TextInputProps = {
+  label: 'Input label',
+  name: 'input',
+}
+
+function renderTextInput(props?: Partial<TextInputProps>) {
+  return render(<TextInput {...defaultProps} {...props} />)
+}
+
+describe('TextInput', () => {
+  it('should render without accessibility violations', async () => {
+    const { container } = renderTextInput()
+
+    expect(
+      //  Ingore the color contrast to avoid an axe-core error cf https://github.com/NickColley/jest-axe/issues/147
```

Comment:
> Ok the 'proper' way to solve this is to turn off the color contrast checks in axe-core which'll never work in a JS Dom environment because visual checks cannot exist.

Ça me fait me poser deux questions.
Il n'y a pas moyen de le configurer globalement ?
Est-ce qu'il n'y a pas un trou dans la raquette ?